SourceS/ssound.h

```diff
@@ -15,4 +15,6 @@ typedef struct SoundSample final {
 	Mix_Chunk *chunk;
 } SoundSample;
 
+void UiErrorOkDialog(const char *text, const char *caption, bool error = true);

```

Comment:
This is in an odd place but I coudn't think of a better one in the moment :/

Reply:
How about adding `SourceS/ui_fwd.h`?